src/api.ts

```diff
@@ -35,6 +37,32 @@ export const bas = {
         return ActionsController.getAction(actionId);
     },
 
+    async getParameter(parameterName: string) : Promise<string | null> {
+        const onBasEnv = "env.WS_BASE_URL";
+        const isInBAS = !_.isEmpty(_.get(process, onBasEnv));
```

Comment:
try using optional chaining here instead. of the `!_.isEmpty(_.get`
- https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html